experimental/packages/opentelemetry-sdk-node/src/TracerProviderWithEnvExporter.ts

```diff
@@ -74,7 +73,23 @@ export class TracerProviderWithEnvExporters extends NodeTracerProvider {
   >([
     ['otlp', () => this.configureOtlp()],
     ['zipkin', () => new ZipkinExporter()],
-    ['jaeger', () => new JaegerExporter()],
+    [
+      'jaeger',
+      () => {
```

Comment:
I would move this into a dedicated function similar as `configureOtlp()` to keep the table short.

Reply:
Good idea :slightly_smiling_face: 
Moved it in [13ee66b](https://github.com/open-telemetry/opentelemetry-js/pull/3739/commits/13ee66b81d4730dbc4cce36685e02ac3313d4209)